libyul/backends/evm/EVMDialect.cpp

```diff
@@ -161,11 +159,9 @@ map<YulString, BuiltinFunctionForEVM> createBuiltins(langutil::EVMVersion _evmVe
 				_assembly.appendConstant(0);
 			else
 			{
-				yulAssert(
-					_context.subIDs.count(dataName) != 0,
-					"Could not find assembly object <" + dataName.str() + ">."
-				);
-				_assembly.appendDataOffset(_context.subIDs.at(dataName));
+				optional<AbstractAssembly::SubID> subId = _context.findSubIDForDataName(dataName);
+				yulAssert(subId,"Could not find assembly object <" + dataName.str() + ">.");
```

Comment:
https://github.com/ethereum/solidity/pull/9047#discussion_r431827298

```suggestion
				yulAssert(subId, "Could not find assembly object <" + dataName.str() + ">.");
```